app/services/arcgis_api/geocoder.rb

```diff
@@ -1,13 +1,12 @@
 module ArcgisApi
   class Geocoder
-    mattr_reader :token, :token_expires_at
-
     Suggestion = Struct.new(:text, :magic_key, keyword_init: true)
     AddressCandidate = Struct.new(
       :address, :location, :street_address, :city, :state, :zip_code,
       keyword_init: true
     )
     Location = Struct.new(:latitude, :longitude, keyword_init: true)
+    API_TOKEN_CACHE_KEY = 'arcgis_api_token'
```

Comment:
do we need the magic `# frozen_string_literal: true` comment or a `.freeze`?

Or we just make this a symbol?

```suggestion
    API_TOKEN_CACHE_KEY = :arcgis_api_token
```